CHANGELOG.md

```diff
@@ -22,6 +22,10 @@
 
     *Jon Rohan*
 
+* Depreacate `FlexItem` component in favor of `BoxComponent`.
```

Comment:
```suggestion
* Deprecate `FlexItem` in favor of `BoxComponent`.
```

Reply:
Also, do you have a plan for when this will be removed? If so, perhaps we could document that here or in the deprecation note?